src/org/kernelci/debian/rootfs.groovy

```diff
@@ -32,25 +32,16 @@ KCI_TOKEN_ID
 - Harcoded paths at jenkins/debian/... for Dockerfile and debos file
```

Comment:
This file should be called RootFS.groovy, or at least start with a capital R to follow the naming convention for classes (like in Java).

Reply:
:+1: